src/screens/HomeScreen/index.tsx

```diff
@@ -26,12 +58,30 @@ const HomeScreen = (): JSX.Element => {
     {
       id: HomeTab.Recents,
       title: strings.screens.recents.title,
-      screen: <RecentsScreen searchText={searchText}></RecentsScreen>,
+      screen: (
+        <RecentsScreen
+          searchText={searchText}
+          isLoading={recentsLoading}
+          recentItems={recentItems}
+          refreshRecentItems={async () => {
```

Comment:
`async-await` is not required here as `refreshRecentItems` is already `async`, therefore you can do:
`refreshRecentItems={refreshRecentItems} `

Reply:
In case that does not fit with correct behavior, you can just use:
`refreshRecentItems={() => refreshRecentItems()}`